blog/gitops-platform-model.md

```
@@ -0,0 +1,302 @@
+title: GitOpsify Cloud Infrastructure with Crossplane and Flux
+sidebar_label: GitOps with Flux and Crossplane
+tags:
+  - Kubernetes
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)